ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -277,6 +278,11 @@ public void setUserAgent(WebView view, @Nullable String userAgent) {
     }
   }
 
+  @ReactProp(name = "mediaPlaybackRequiresUserAction")
+  public void setMediaPlaybackRequiresUserAction(WebView view, boolean requires) {
+    view.getSettings().setMediaPlaybackRequiresUserGesture(requires);
```

Comment:
> I don't think getSettings does return null

Can you check the source code to be 100% sure? Also the behavior can change across Android versions. It's better to always check for `null` to be safe. 

See https://en.wikipedia.org/wiki/Defensive_programming